tests/test_hubspot_bookmarks.py

```diff
@@ -30,11 +30,12 @@ def name():
 
     def streams_to_test(self):
         """expected streams minus the streams not under test"""
-        expected_streams = self.expected_streams().difference(STREAMS_WITHOUT_CREATES)
+        """expected_streams = self.expected_streams().difference(STREAMS_WITHOUT_CREATES)
 
         return expected_streams.difference({
             'subscription_changes', # BUG_TDL-14938 https://jira.talendforge.org/browse/TDL-14938
-        })
+        })"""
+        return {'contact_lists'}
```

Comment:
I think this was just during testing to only test contact_lists but shouldn't be in the final version

Reply:
Removed